src/ir/types/proto/entity_type.h

```diff
@@ -16,21 +16,24 @@
 #ifndef SRC_IR_TYPES_PROTO_ENTITY_TYPE_H_
 #define SRC_IR_TYPES_PROTO_ENTITY_TYPE_H_
 
+#include "src/ir/proto/raksha_type.pb.h"
 #include "src/ir/types/entity_type.h"
 #include "src/ir/types/type_factory.h"
-#include "third_party/arcs/proto/manifest.pb.h"
 
 namespace raksha::ir::types::proto {
 
+using type::proto::EntityTypeProto;
```

Comment:
I think we might want to get rid of `EntityType`s and `Schema`s altogether. We aren't using them in any significant way, and we might want to build something different for Raksha's purposes.

Reply:
Let us keep them around for now. This would become useful/needed when we start talking about SQL tables and their schemas.

Reply:
Acknowledge